### PR TITLE
Enable history in bacon shell

### PR DESCRIPTION
**Pull request by rnc**
### Checklist:

* [ ] Have you added a note in the [CHANGELOG wiki](https://github.com/project-ncl/bacon/wiki/Changelog) for your change if user-facing?
* [ ] Have you added unit tests for your change?

### CODE REVIEW

**Comment by rnc** [comment]
@michalszynkiewicz Added your shell history for you. Couldn't see a way of enabling that extra prompt by default so added a log line for now.

File: cli/src/main/java/org/jboss/pnc/bacon/cli/App.java [hunk 1]
```diff
@@ -143,17 +143,16 @@ public int run(String[] args) {
                         .terminal(terminal)
                         .completer(systemRegistry.completer())
                         .parser(parser)
+                        .variable(LineReader.HISTORY_FILE, Constant.HISTORY)
+                        .variable(LineReader.HISTORY_FILE_SIZE, 50)
```
Comment (michalszynkiewicz) [inline cli/src/main/java/org/jboss/pnc/bacon/cli/App.java:147]: is that last 50 entries in the history file? I'd like much more :)

**Review by michalszynkiewicz — APPROVED** [review]

**Comment by michalszynkiewicz** [comment]
Thanks a lot @rnc !

File: cli/src/main/java/org/jboss/pnc/bacon/cli/App.java (hunk 1 above)
Comment (rnc) [reply cli/src/main/java/org/jboss/pnc/bacon/cli/App.java:147]: Er, sure, sorry just picked a random number :-)

**Pull request merged**